feat(performance): removes fid note and update webvitals intro to clarify field data (#10437)
getsentry/sentry-docs: docs/product/insights/web-vitals/index.mdx

<Include name="feature-available-for-plan-business.mdx" />

Web vitals are a set of metrics that measure the quality of the user experience on a web page. To learn more about these metrics, see [Web Vitals Concepts](/product/insights/web-vitals/web-vitals-concepts/).

> In Sentry, web vitals are gathered from real user traffic with [supported browsers](web-vitals-concepts/#browser-support), on your instrumented web apps. These web vitals are used to calculate a [Performance Score](#performance-score) for your web application. Using these metrics, we surface the pages that have the most [opportunity](#opportunity) to improve your app's overall performance.

Reviewer: ```suggestion
For your instrumented web apps, Sentry gathers web vitals from real user traffic through [supported browsers](web-vitals-concepts/#browser-support). These web vitals are used to calculate a [Performance Score](#performance-score) for your web application. Using these metrics, we surface the pages that have the most [opportunity](#opportunity) to improve your app's overall performance.
```
